tests/apps/populate_reentry/models.py

```diff
@@ -0,0 +1 @@
+from .user_script import some_value  # NOQA
```

Comment:
It could help to add a comment like `This import triggers a first/second? call to django.setup()`.